src/usexp/usexp.ml

```diff
@@ -115,6 +142,17 @@ let rec pp_split_strings ppf = function
         Format.fprintf ppf "%S" s
     end else
       Format.pp_print_string ppf s
+  | Quoted_string s ->
```

Comment:
We should factorize this code with the `Atom` case into a separate function. For instance, I just realized that I made a mistake Atom case, the `Format.fprintf ppf "%S" s` doesn't escape the string according to S-expressions conventions...

Reply:
I'll do that but I'm not too sure the output format `pp_split_strings` is targeting.  Maybe adding a test somewhere would be good.

Reply:
@diml Just pushed the updated version of `Usexp`.

Reply:
It's for `jbuilder rules`, so that long strings in rules are displayed nicely. A test would be nice indeed